In [1]:
import pandas as pd
import numpy as np

In [15]:
df=pd.read_csv('data/rental_price.csv')

In [14]:
a=df['price'].str.replace('$','',regex=False)

print(a)


0       300.00
1        99.00
2       100.00
3       100.00
4       599.00
         ...  
5829    125.00
5830    179.00
5831     85.00
5832     25.00
5833    120.00
Name: price, Length: 5834, dtype: object


Online Tourist Agency   
https://www.xotels.com/en/glossary/independent-hotel#:~:text=An%20Independent%20Hotel%20is%20a,chain%20are%20considered%20'independent'.     


### Introduction


OYO hotels ,as known as OYO hotels and homes.In recent years, OYO hotels have developed rapidly in China, mainly in franchise and chain of lease, occupying many second and third-tier cities' market.  
This report try to predit the rental price of the OYO hotels in China.So the target variable is "rental price",and it's a regression problem.   
Because OYO's market layout is concentrated in second and third-tier cities, it is also the main city in China, occupying a large part of economic development. Therefore, the research on it have a great reference value.    
This report download the dataset from Kaggle[1],but all the infomation can be gathered form Chinese OTA platforms.This dataset has 5834 data points and 25 features,



### EDA

### Data preprocessing 

### References

### Github